docker-compose.yml

```diff
@@ -155,6 +155,17 @@ services:
       - DISABLED_FEATURES=["downloadEndpoint","functionEndpoint","pdfEndpoint","screencastEndpoint","scrapeEndpoint","statsEndpoint","workspaces"]
       - FUNCTION_ENABLE_INCOGNITO_MODE=true
 
+  translation:
```

Comment:
Would be nice to prevent this service from starting automatically. This can be achieved with profiles:
https://docs.docker.com/compose/profiles/